src/components/Navbar.jsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
 import Navbar from 'react-bootstrap/Navbar';
-import './index.css';
+import '../index.css';
 import { Container, Nav, FormControl, Form } from 'react-bootstrap';
-
-
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
 const Navigationbar = () => {
   return (
-    <Navbar >
+    <Navbar>
       <Container fluid>
         <Navbar.Brand href='#'>Food ordering app</Navbar.Brand>
         <Navbar.Collapse id='navbarScroll'>
-          <Nav className='me-auto my-2 my-lg-0' style={{ maxHeight: '100px' }} >
-          </Nav>
+          <Nav
+            className='me-auto my-2 my-lg-0'
+            style={{ maxHeight: '100px' }}
+          ></Nav>
           <Form className='d-flex'>
```

Comment:
Adding `align-items-center` as a class should fix the alignment issue of the icon.